extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/model/ContractAgreementTransfer.java

```diff
@@ -35,8 +34,8 @@ public class ContractAgreementTransfer {
     private String transferProcessId;
     @Schema(description = "Last Change Date", requiredMode = Schema.RequiredMode.REQUIRED)
     private OffsetDateTime lastUpdatedDate;
-    @Schema(description = "Current Status", requiredMode = Schema.RequiredMode.REQUIRED)
-    private TransferProcessStates state;
+    @Schema(description = "Current State", requiredMode = Schema.RequiredMode.REQUIRED)
+    private TransferStateInfo state;
```

Comment:
In the TransferSimplifiedState.java I read the following: `This class exists because we want to highlight either running or failed transfer processes in our UI`

Hence my question: why not be more clear in the Class Name => TransferProcessSimplifiedState, TransferProcessStateInfo, etc